js/menu.js

```diff
@@ -0,0 +1,32 @@
+let menuData = new XMLHttpRequest();
+menuData.open("GET", "./api/menu.json");
+menuData.send();
+menuData.addEventListener("load", menu);
+
+//Load menu, page title, and page heading dynamically from menu.json
+function menu(){
```

Comment:
Give more appropriate function and variable names. eg: instead of _menu_ give the function name as _loadMenuDetails_